ci/deps/travis-37.yaml

```diff
@@ -13,8 +13,8 @@ dependencies:
   - pyarrow
   - pytz
   # universal
-  - pytest>=4.0.2
-  - pytest-xdist
+  - pytest>=5.0.0
```

Comment:
why 5.0.0 in some places and 5.0.1 in others?

Reply:
Just sloppiness on my part.